annual-calendar-creator/public/calendarsettings.html

```diff
@@ -0,0 +1,48 @@
+<html>
+    <head>
+        <link type="text/css" rel="stylesheet" href="./calendarsettings.css">
+    </head>
+    <body>
+
+        <script type="text/javascript">
+            function setCalendarSettings() {
+                let year = document.getElementById('calendar-year').value;
+                let language = document.getElementById('calendar-language').value;
+                
+                let localizedMonths = [1,2,3,4,5,6,7,8,9,10,11,12].map(m => new Intl.DateTimeFormat(language,{month: 'long', year: 'numeric'}).format(new Date(`${m}/01/${year}`)));
```

Comment:
I think `new Date(`${m}/01/${year}`)` would differ depending on the browser locale, so would `new Date(year, m - 1)` be safer to get the date here?

Reply:
Adjusted this, good catch!